.github/workflows/build-rock.yaml

```diff
@@ -10,7 +10,12 @@ on:
 
 jobs:
   build-rock:
-    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        runner:
+          - [self-hosted, amd64]
+          - [self-hosted, arm64]
```

Comment:
I'm taking this PR as an opportunity to build and publish in a trusted environment.